cpu/esp_common/esp-wifi/esp_wifi_netdev.c

```diff
@@ -947,20 +947,20 @@ void esp_wifi_setup (esp_wifi_netdev_t* dev)
 
 #if defined(MODULE_ESP_WIFI_ENTERPRISE) && !defined(MODULE_ESP_WIFI_AP)
 
-#ifdef ESP_WIFI_EAP_ID
-    esp_wifi_sta_wpa2_ent_set_identity((const unsigned char *)ESP_WIFI_EAP_ID,
-                                       strlen(ESP_WIFI_EAP_ID));
-#endif /* ESP_WIFI_EAP_ID */
-#if defined(ESP_WIFI_EAP_USER) && defined(ESP_WIFI_EAP_PASS)
+#ifdef WIFI_EAP_ID
```

Comment:
We should have some default definitions here for compatibility reasons:
```suggestion

#if !defined(WIFI_EAP_ID) && defined(ESP_WIFI_EAP_ID)
#define WIFI_EAP_ID     ESP_WIFI_EAP_ID
#endif

#if !defined(WIFI_EAP_USER) && defined(ESP_WIFI_EAP_USER)
#define WIFI_EAP_USER   ESP_WIFI_EAP_USER
#endif

#if !defined(WIFI_EAP_PASS) && defined(ESP_WIFI_EAP_PASS)
#define WIFI_EAP_PASS   ESP_WIFI_EAP_PASS
#endif

#ifdef WIFI_EAP_ID
```